local/compose/create.go

```diff
@@ -296,6 +296,32 @@ func buildContainerMountOptions(p types.Project, s types.ServiceConfig, inherit
 		}
 		mounts = append(mounts, mount)
 	}
+	for _, secret := range s.Secrets {
+		if contains(inherited, secret.Target) {
```

Comment:
doing so, IIUC, will prevent changing the secret file name in compose file and get sercet bind updated on `up`.
IMHO inherited volumes from previous container should be default be overriden by config.
Not sure how docker-compose handle this, we should align

Reply:
+1 to align with docker-compose behaviour (and indeed I would expect to override with config in case of clash)